scheduler-interface-spec.md

```diff
@@ -792,6 +792,31 @@ message ForgotAllocation {
 }
 ```
 
+#### Event Cache
```

Comment:
Thanks for the docs, I think we need another JIRA to add doc like this to explain rest of plugins and messages.
Just created https://issues.apache.org/jira/browse/YUNIKORN-190 for this.

Reply:
Thanks @yangwwei !